api/buckets/service.go

```diff
@@ -388,6 +388,7 @@ func (s *Service) PushPath(server pb.API_PushPathServer) error {
 		return err
 	}
 
+	size := <-chSize
```

Comment:
Tricky one. `size` was being used in `pb.PushPathReply_Event` but was not being waited correctly for the progress goroutine. This resulted in some garbage size value.

Reply:
Nice, good catch 💪